samples/apps/spring-petclinic/patch/pom.xml.patch

```diff
@@ -14,17 +14,17 @@ index fc907e4..cfeb8ff 100644
      <dependency>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-test</artifactId>
-@@ -261,6 +266,14 @@
+@@ -253,6 +258,14 @@
          <enabled>false</enabled>
        </snapshots>
      </repository>
 +    <repository>
-+      <snapshots>
-+        <enabled>true</enabled>
-+      </snapshots>
 +      <id>spring-release</id>
 +      <name>Spring Release</name>
 +      <url>https://repo.spring.io/release</url>
++      <snapshots>
++        <enabled>false</enabled>
++      </snapshots>
 +    </repository>
```

Comment:
why do we need this change in this PR?

Reply:
this shouldn't be part of this PR, I missed that accident addition, thx - will remove, thx